packages/kit/src/runtime/client/start_embedded.js

```diff
@@ -0,0 +1,38 @@
+import { DEV } from 'esm-env';
+
+let instance_id = -1;
+
+// This is a hack to get the client module url while making sure Vite compiles it correctly.
+const modules = import.meta.glob('./client.js');
+const importPathRegex = /import\(['"]([^'"]+)['"]\)/;
+const client_import = Object.values(modules)[0];
+const importStatement = client_import.toString();
+const match = /** @type {RegExpMatchArray } */ (importStatement.match(importPathRegex));
+const client_url = match[1];
+
+/**
+ * @param {import('./types.js').SvelteKitApp} app
+ * @param {HTMLElement} target
+ * @param {any} [hydrate] (see _hydrate in client.js for the type)
+ */
+export async function start(app, target, hydrate) {
+	if (DEV && target === document.body) {
+		console.warn(
+			'Placing %sveltekit.body% directly inside <body> is not recommended, as your app may break for users who have certain browser extensions installed.\n\nConsider wrapping it in an element:\n\n<div style="display: contents">\n  %sveltekit.body%\n</div>'
+		);
+	}
+
+	// In embedded mode, the client is potentially loaded multiple times for different embeddings.
```

Comment:
should we just always do this? It might simplify things by allowing us just to have a single start script

Reply:
The embedded version creates an extra chunk and might be a bit slower because of the dynamic import. As such I think it's better to have a separate version.